Production/python/Production.py

```diff
@@ -226,6 +244,8 @@
 
 if isPhase2:
     process.p.insert(0, process.slimmedElectronsMerged)
+elif isUltraLegacy:
+    pass
 else:
     process.p.insert(2, process.boostedSequence)
```

Comment:
boostedSequence should be rerun only for run 2 pre-UL, better to have `isRun2PreUL` flag and do a separate if.
```python
if isPhase2:
    process.p.insert(0, process.slimmedElectronsMerged)
if isRun2PreUL:
   process.p.insert(2, process.boostedSequence)
```